Cargo.toml

```diff
@@ -149,7 +149,7 @@ toml = { version = "0.8.19", default-features = false, features = ["display", "p
 tonic = { version = "0.11", default-features = false, features = ["transport", "codegen", "prost", "tls", "tls-roots", "gzip"] }
 tonic-build = { version = "0.11", default-features = false, features = ["transport", "prost"] }
 uuid = { version = "1.10.0", features = ["v4", "v7", "serde"] }
-vrl = { version = "0.19.0", features = ["arbitrary", "cli", "test", "test_framework"] }
+vrl = { git = "https://github.com/vectordotdev/vrl", branch = "main", features = ["arbitrary", "cli", "test", "test_framework"] }
```

Comment:
Do we have any mitigation for the following scenario?

1. Vector builds successfully with the latest VRL main
2. We merge a new commit that changes an interface in VRL
3. Vector builds will fail until we update the code

Reply:
The version will still be locked so an update to VRL won't break the Vector build on `master`. Instead, a PR opened by dependabot that bumps VRL to latest will fail at which point we can intervene to make the necessary Vector updates in the same PR.